react-frontend/src/components/products/assembling.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import { Col, Row, Grid } from 'react-flexbox-grid';
+import ProductsCard from './productscard';
+
+const Assembling = () => {
+  let cardOne = (
```

Comment:
I believe these should all be 'const' not 'let'

Reply:
Also, I'm not sure of this pattern. If the form of the card is always the same we should pass the fields in as props, not the entire HTML as a prop.

Reply:
II'll make some const to define the content within each and create a loop to render, there are hard coded links or bold text within the content so it needs to be hard coded